config/crd/bases/oadp.openshift.io_dataprotectionapplications.yaml

```diff
@@ -595,10 +595,10 @@ spec:
                             nodeSelector:
                               additionalProperties:
                                 type: string
-                              description: nodeSelector defines the nodeSelector to be supplied to Restic podSpec
+                              description: nodeSelector defines the nodeSelector to be supplied to Velero podSpec
```

Comment:
Thanks @sbahar619 
The CRDs are created by `make manifests` command, so only changing them would not persist the changes. They are generated from the files in `api/v1alpha1/` folder. For example, this line was generated from [this one](https://github.com/openshift/oadp-operator/blob/master/api/v1alpha1/oadp_types.go#L118). Can you please apply these changes to that file and run `make manifests` instead?

Reply:
Sure, thanks for letting me know @mateusoliveira43 
After the make manifests command, the config/crd/bases/oadp.openshift.io_dataprotectionapplications.yaml file is also being modified.
Should I also include those changes?

Reply:
Yeah, I think there is no CI automation for generating them for us (team, correct me if I am wrong :smile:)

Reply:
OK, I pushed the changes.
Because it using a generic struct type (PodConfig) for Velero and Restic, I just removed the reference for Restic and made the description generic.

Reply:
OK, thanks!

Reply:
Run `make bundle`

Reply:
My bad @sbahar619, it was `make bundle` instead of `make manifests`. @kaovilai have run it :smile: